### PR TITLE
[PD-1358] null=True -> default=''

### PR DESCRIPTION
**Pull request by aspidites**
Tests are failing locally, but only for me, so I wanted this on jenkins to see if I could reproduce the errors. That said, the seo branch fails for me unconditionally, regardless of which branch I'm on, so I assume something about my local database isn't identical to production/qc/staging databases.

### CODE REVIEW

File: mypartners/models.py [hunk 1]
```diff
@@ -843,7 +843,7 @@ class ContactLogEntry(models.Model):
     contact_identifier = models.CharField(max_length=255)
     content_type = models.ForeignKey(ContentType, blank=True, null=True)
     delta = models.TextField(blank=True)
-    object_id = models.TextField('object id', blank=True, null=True)
+    object_id = models.TextField('object id', blank=True, default='')
```
Comment (tdphillips) [inline mypartners/models.py]: I know you didn't add this, but I kind of have to wonder why an id is a textfield.

Comment (almathew) [inline mypartners/models.py]: https://github.com/django/django/blob/master/django/contrib/admin/models.py#L32

Comment (almathew) [inline mypartners/models.py]: I think the longer answer is that the id field doesn't have to be the typical autoincrement field, and textfield is a happier catch-all even in the worst cases solution.

Comment (tdphillips) [inline mypartners/models.py]: Makes sense.

**Comment by aspidites** [comment]
Abandoning this.

**Pull request closed without merging**